contact.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rylie Butner | Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+              <li><a href="index.html"><h2>Rylie Butner</h2></a></li>
+              <li><a href="work.html">Work</a></li>
+              <li><a href="misc.html">Misc.</a></li>
+              <li><a href="contact.html">Contact</a></li>
+            </ul>  
+        </nav>
+    </header>
+    <main>
+        <h1>Contact Form</h1>
+    <form class="cf">
```

Comment:
We will go into forms later in the semester. Labels are required so keep that in mind

Reply:
Got it, I've added labels.